explorer/interpreter/value_transform.h

```diff
@@ -179,13 +180,13 @@ class ValueTransform : public TransformBase<Derived> {
 
   // Preserve vtable during transformation.
   auto operator()(Nonnull<const VTable* const> vtable)
-      -> Nonnull<const VTable* const> {
+      -> Nonnull<const VTable*> {
```

Comment:
That `const` was more for semantics than anything, to convey that the pointers are not meant to change, but I remember the linter being annoyed by it, and probably OK to drop :)

Reply:
TBH I assumed this was a straight copy-paste of the member type, where the `const` has the effect you describe.

For reference, this is https://clang.llvm.org/extra/clang-tidy/checks/readability/const-return-type.html

> Checks for functions with a const-qualified return type and recommends removal of the const keyword. Such use of const is usually superfluous, and can prevent valuable compiler optimizations. Does not (yet) fix trailing return types.